cocinaborcelle/src/clientApi/restaurantes/restaurantType.ts

```diff
@@ -0,0 +1,9 @@
+import { Localizacao, RedeSocial, Restaurante } from "@prisma/client"
+
+type RestauranteType = {
+    restaurante: Restaurante,
+    redesocial: RedeSocial,
+    localizacao: Localizacao,
+}
```

Comment:
```suggestion
type RestauranteType = {
    redesocial: RedeSocial,
    localizacao: Localizacao[]
} & Restaurante
```

esse é o type do que tá retornando da api do lado do servidor. Para saber como ver isso, é só ir lá em api/restaurante/[slug].ts e passar o mouse por cima do que vc tá passando no json